test/powershell/Modules/Microsoft.PowerShell.Management/Get-Content.Tests.ps1

```diff
@@ -163,5 +163,82 @@ Describe "Get-Content" -Tags "CI" {
   ) {
     param($cmdlet)
     (Get-Command $cmdlet).Parameters["stream"] | Should BeNullOrEmpty
-  }  
+  }
+ 
+  It "Should return no content when an empty path is used with -Raw switch" {
+    Get-ChildItem $TestDrive -Filter "*.raw" | Get-Content -Raw | Should Be $null
+  }
+
+  It "Should return no content when -TotalCount value is 0" {
+	  Get-Content -Path $testPath -TotalCount 0 | Should Be $null
+  }
+
+  It "Should throw TailAndHeadCannotCoexist when both -Tail and -TotalCount switches are used" {
+    try {
+      Get-Content -path $testPath -Tail 1 -TotalCount 1 -ea stop
```

Comment:
Please fix formatting "-path"  -> "-Path", "stop" -> "Stop".

Please use full name for cmdlets, parameters and values.    "-ea" -> "ErrorAction".

Below too.